src/sites/image/imgtorrnt.in.js

```diff
@@ -0,0 +1,13 @@
+$.register({
+  rule: {
+    host: /^imgtorrnt.in$/,
+    path: /^\/view\.php$/,
+    query: /^\?id=.*/,
+  },
+  ready: function () {
+    'use strict';
+
+    let img = $('center div table.tg tbody tr td center img');
```

Comment:
ES6 is good, but not all supported platform supported `let`. (e.g.: PaleMoon or Chrome < 41)